tests/ssl-opt.sh

```diff
@@ -2941,6 +2932,9 @@ run_test    "Max fragment length DTLS: disabled, larger message" \
             -S "Maximum outgoing record payload length is 16384" \
             -c "fragment larger than.*maximum "
 
+# Make sure it was compiled with lengths over 4096
+requires_config_value_at_least "MBEDTLS_SSL_IN_CONTENT_LEN" 4096
+requires_config_value_at_least "MBEDTLS_SSL_OUT_CONTENT_LEN" 4096
```

Comment:
In practice, we always test `MBEDTLS_SSL_IN_CONTENT_LEN` and `MBEDTLS_SSL_OUT_CONTENT_LEN` together. Maybe write an auxiliary function and write e.g. `requires_max_content_len 4096`?